app/assets/stylesheets/_alerts.scss

```diff
@@ -0,0 +1,88 @@
+@import 'base/globals';
+
+.alert {
+  margin: 1rem 5rem;
+  padding: 1rem 1.25rem;
+  border: 1px solid transparent;
```

Comment:
Avoid `px` usage and always go with `rem`. While modern browsers can scale `px` _(thus the premise of respect user preference is getting old)_ it is still a good practice to unify the units we use to define styles to avoid questions and confusion with new developers.

Reply:
for borders, I don't think it makes sense to use rems. Borders width shouldn't change with screen size and are meant to be fixed. Also, browsers have problems rendering sub-pixel borders https://chenhuijing.com/blog/about-subpixel-rendering-in-browsers.

I do agree to have these for margin and padding tho.